src/Calculator/Views/Calculator.xaml

```diff
@@ -19,20 +18,20 @@
             <TextBlock Margin="2,0,0,0"
                        Foreground="{ThemeResource SystemControlPageTextBaseMediumBrush}"
                        IsTextScaleFactorEnabled="False"
-                       Text="{Binding Token, Mode=OneWay}"/>
+                       Text="{x:Bind Token}"/>
```

Comment:
These bindings were updated to be `OneTime` (default mode for `x:Bind`).  There are places in the codebase where the `Token` property is updated and those changes wouldn't show in the UI after this change.  Are you sure this is correct?

Reply:
The only place I found where Token is modified is a dead code (the UWP Calculator doesn't allowing users to edit an operation contrary to the Win7 version) but while this dead code is still part of the repository, you are right, we should should keep `Mode=OneWay`.

Reply:
We could also remove the dead code 👍  Then this can be kept as a `OneTime`